testnet/testnet-deploy-contracts.sh

```diff
@@ -59,7 +59,11 @@ network_cfg='{
             "url" : '"\"http://${l1host}:${l1port}\""',
             "live" : false,
             "saveDeployments" : true,
-            "deploy": [ "deployment_scripts/layer1", "deployment_scripts/testnet" ],
+            "deploy": [ 
```

Comment:
Can there be a comment stating where the network cfg flags can be read / described ?